pkg/serviceaccount/legacy.go

```diff
@@ -40,7 +48,11 @@ func LegacyClaims(serviceAccount v1.ServiceAccount, secret v1.Secret) (*jwt.Clai
 		}
 }
 
-const LegacyIssuer = "kubernetes/serviceaccount"
+const (
+	LegacyIssuer          = "kubernetes/serviceaccount"
+	LastUsedAnnotationKey = "kubernetes.io/legacy-token-last-used"
```

Comment:
I'm trying to remember why we said we'd add both an annotation and a label... the label makes sense so you can select token secrets with it... I can't think of a use case for an annotation with the same data... do you remember?

Reply:
initially we are only trying to add annotation. label is something optional added for query which was just a boolean value. during the iteration, their values merged and there is no need to have them both. done.